x-pack/legacy/plugins/remote_clusters/public/app/sections/components/remote_cluster_form/remote_cluster_form.js

```diff
@@ -500,18 +498,62 @@ export class RemoteClusterForm extends Component {
   }
 
   renderErrors = () => {
-    const { areErrorsVisible } = this.state;
+    const {
+      areErrorsVisible,
+      fields: {
+        name,
+      },
+      fieldsErrors: {
+        name: errorClusterName,
```

Comment:
We also need to read the errors for the seeds:

```js
      fieldsErrors: {
        name: errorClusterName,
        seeds: errorsSeeds,
      },
```

Reply:
@myasonik I don't believe this is resolved yet. The seed errors come from two places, one of which is handled by the current changes but the other one would be handled by this suggestion and my suggestion in the next comment.

Reply:
Sorry, I've been marking stuff as resolved as I work on it locally but I haven't pushed the changes back up as I'm not done with everything quite yet.

So I guess you only mark as resolved after you push changes at Elastic? (Still pretty new here 🙂)

Reply:
Closing for now as I believe my latest commit resolves this.

Reply:
Ah, thanks for explaining! I understand now, I guess I just need to think of "resolved" as "resolved by the author" (duh). 😄

Reply:
No, looks like I was in the wrong! I heard from others that Elastic tends to prefer letting the reviewer resolve conversations so they can make sure their issues were addressed.